### PR TITLE
feature: add distributed lock interface

### PR DESCRIPTION
**Pull request by a364176773**

### CODE REVIEW

File: common/src/main/java/io/seata/common/Constants.java [hunk 1]
```diff
@@ -131,4 +131,30 @@
...
+
+    /**
+     * The constant TX_TIMEOUT_CHECK
+     */
+    String TX_TIMEOUT_CHECK = "TxTimeoutCheck";
+
+    /**
+     * The constant UNDOLOG_DELETE
+     */
+    String UNDOLOG_DELETE = "UndoLogDelete";
```
Comment (xingfudeshi) [inline common/src/main/java/io/seata/common/Constants.java:158]: UndologDelete

**Review by caohdgege — APPROVED** [review]
LGTM

File: common/src/main/java/io/seata/common/Constants.java (hunk 1 above)
Comment (a364176773) [reply common/src/main/java/io/seata/common/Constants.java:158]: ok

**Review by slievrly — APPROVED** [review]
LGTM,  need to further define how to load different lock implementations

**Review by l81893521 — APPROVED** [review]
LGTM

**Pull request merged**